src/Toolbox.php

```diff
@@ -3006,6 +3006,35 @@ public static function getFgColor(string $color = "", int $offset = 40, bool $in
         return "#" . $fg_color;
     }
 
+
+    public static function isColorDark(string $color = ""): bool
```

Comment:
This parameter is mandatory, it should not have a default value.

```suggestion
    public static function isColorDark(string $color): bool
```